### PR TITLE
Improvement Localizable.strings

### PR DESCRIPTION
**Pull request by holton-jiang**
"saveToOmnivore"、"filterGeneric" and "texttospeechBetaWaitlist".

### CODE REVIEW

File: apple/OmnivoreKit/Sources/Views/Resources/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -122,10 +122,10 @@
 // Welcome View
 "welcomeTitle" = "认真的读者所喜爱的稍后阅读。";
 "welcomeLearnMore" = "了解更多";
-"welcomeSignupAgreement" = "注册即表示您同意 Omnivore’s\n";
+"welcomeSignupAgreement" = "注册即表示您同意 Omnivore\n";
```
Comment (sywhb) [inline apple/OmnivoreKit/Sources/Views/Resources/zh-Hans.lproj/Localizable.strings:125]: “您使用我们的服务即表示您已同意本条款” might be better

**Review by sywhb — COMMENTED** [review]
All the translations look good except for one

**Comment by holton-jiang** [comment]
> All the translations look good except for one

Is it the voice one?

How about “朗读” instead?

File: apple/OmnivoreKit/Sources/Views/Resources/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (sywhb) [reply apple/OmnivoreKit/Sources/Views/Resources/zh-Hans.lproj/Localizable.strings:125]: @holton-jiang I meant this one

**Comment by jacksonh** [comment]
I am going to merge this one and then update that single string.

**Comment by holton-jiang** [comment]
> I am going to merge this one and then update that single string.

Thank you.

**Pull request merged**